### PR TITLE
Add -kubeconfig flag

### PR DESCRIPTION
**Pull request by johngmyers**

### CODE REVIEW

**Comment by johngmyers** [comment]
@samba would something like this work for you?

**Comment by samba** [comment]
@johngmyers Seems sound to me. 👍

File: main.go [hunk 1]
```diff
@@ -61,7 +63,7 @@ func main() {
 
 	// Create a Kubernetes client.
 	// Initialize a configuration based on the default service account.
```
Comment (raghu67) [inline main.go:65]: Nit pick comment. It would not be default service account if a custom kubeconfig is passed in.

**Review by raghu67 — APPROVED** [review]
I have added a minor comment inline.

**Comment by johngmyers** [comment]
This is in the image proofpoint/certificate-init-container:0.3.0

**Pull request merged**